inference-engine/src/mkldnn_plugin/nodes/mkldnn_ctc_greedy_decoder_node.h

```diff
@@ -15,12 +15,13 @@ class MKLDNNCTCGreedyDecoderNode : public MKLDNNNode {
 
     void getSupportedDescriptors() override {};
     void initSupportedPrimitiveDescriptors() override;
-    void createPrimitive() override {};
+    void createPrimitive() override;
     void execute(mkldnn::stream strm) override;
     bool created() const override;
+    void executeDynamicImpl(dnnl::stream strm) override;
+    void prepareParams() override;
```

Comment:
override `needPrepareParams` for the function to return false
instead `prepareParams`